pages/main-delete.js

```diff
@@ -15,15 +15,18 @@ function view (state, prev, send) {
 
   const header = Header({
     create: () => send('repos:create'),
-    download: (link) => send('repos:download', link)
+    download: (link) => send('repos:clone', link)
   })
 
   return html`
     <body>
       ${sprite()}
       ${header}
       ${Table(archives, send)}
-      ${confirmModal(() => send('repos:deleteConfirm', link))}
+      ${confirmModal(() => send('repos:remove', {
+        confirmed: true,
+        key: link
```

Comment:
👍 I'm guilty of this too in the CLI.

In my mind `key` the 64 character hash (mostly as a buffer), without any other characters. `link` is any think that contains a key (or references one), e.g. `dat://<key>`, `datproject.org/view/<key>`, `datproject.org/karissa/more-tweets-more-votes` (which resolves a key via API). Though I mostly refer to all of these as `key` in the CLI.

But we should decide on a [consistent terminology](https://docs.datproject.org/terms#dat-link).